### PR TITLE
[SCB-1815] Support sdk register on local SC and discover on SCA

### PR DESCRIPTION
**Pull request by heyile**
…y and change outlog item name

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by heyile** [comment]
**modify as bellow**  :

* support use differently address to register and discover

```yaml

# keep same with go chassis
# 
servicecomb:
  service:
    registry:
      address: xxx:30100,xxx:30100
      registrator:
        # registry 
        address: xxx:3100,xxx:30100
      serviceDiscovery:
        # discovery
        address: xxx:3100,xxx:30100


```

* change name of access log item

* remove readme.md of accesslog

File: common/common-access-log/src/main/java/org/apache/servicecomb/common/accessLog/core/element/impl/DatetimeAccessConfigurableItem.java [hunk 1]
```diff
@@ -49,15 +49,15 @@
...
+  public DatetimeAccessConfigurableItem() {
     this(DEFAULT_DATETIME_PATTERN);
   }
 
   /**
    * the configurations not specified will get a default value.
    * @param config the format of configuration is "PATTERN|TIMEZONE|LOCALE" or "PATTERN". It depends on whether the config contains the separator "|"
    */
-  public DatetimeConfigurableItemAccess(String config) {
+  public DatetimeAccessConfigurableItem(String config) {
```
Comment (liubao68) [inline common/common-access-log/src/main/java/org/apache/servicecomb/common/accessLog/core/element/impl/DatetimeAccessConfigurableItem.java]: ConfigurableDatetimeAccessItem

Comment (heyile) [reply common/common-access-log/src/main/java/org/apache/servicecomb/common/accessLog/core/element/impl/DatetimeAccessConfigurableItem.java]: ok, i will modify it in #1665

File: core/src/main/java/org/apache/servicecomb/core/bootup/AddressInformationCollector.java [hunk 2]
```diff
@@ -18,15 +18,21 @@
...
 public class AddressInformationCollector implements BootUpInformationCollector {
   @Override
   public String collect() {
-    return "Service Center: "
-        + getCenterInfo(Deployment.getSystemBootStrapInfo("ServiceCenter"))
-        + "\n" + "Config Center: "
-        + getCenterInfo(Deployment.getSystemBootStrapInfo("ConfigCenter"));
+    return "Service Center:\n"
+        + "  Registration: "
+        + getCenterInfo(Deployment.getSystemBootStrapInfo(DeploymentProvider.SYSTEM_KEY_SERVICE_CENTER_REGISTRY))
```
Comment (liubao68) [inline core/src/main/java/org/apache/servicecomb/core/bootup/AddressInformationCollector.java]: not correct, lack of servicecomb.service.registry.address

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfigBuilder.java [hunk 3]
```diff
@@ -129,6 +131,46 @@ public boolean isSsl() {
     return ipPortList;
   }
 
+  public ArrayList<IpPort> getRegistryIpPort() {
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfigBuilder.java]: This 3 methods and atract to one `public ArrayList<IpPort> getIpPort(String key) `

**Pull request closed without merging**